torch/fx/node.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, graph: 'Graph', name: str, op: str, target: Target,
         # does not produce a value, it's more of a notation. Thus, this value
         # describes the type of args[0] in the `return` node.
         self.type : Optional[Any] = type
+        # Attrbutes that are needed to lowering the fx node.
+        # This dict is filled by param_fetch.
+        self.attrs_for_lowering : Dict[str, Any] = {}
```

Comment:
Please don't add this code to FX core, it is not generally applicable. You can either monkey-patch this attribute, or create a subclass of `Node` to inject into your `Graph`

Reply:
Ah I see, I'll try monkey-patch first (hopefully no mypy error